extensions/shlink/src/shorten-url.tsx

```diff
@@ -0,0 +1,216 @@
+        })();
+    }, []);
+
+    useEffect(() => {
+        if (domainsData) {
+            setValue("domain", domainsData?.find((domain) => domain.isDefault).domain);
+        }
+    }, [domainsData]);
+
+    return (
+        <Form
+            searchBarAccessory={<Form.LinkAccessory text="Web Dashboard" target="https://app.shlink.io/" />}
+            isLoading={isDomainsLoading || isTagsLoading || isLoading}
+            actions={
+                <ActionPanel>
+                    <Action.SubmitForm icon={Icon.CopyClipboard} title="Submit and Copy" onSubmit={handleSubmit} />
+                </ActionPanel>
+            }
+        >
+            <Form.TextField title="Long URL" storeValue={false} autoFocus {...itemProps.longUrl} />
+            <Form.Separator />
+            <Form.TextField
+                title="Title"
+                info="Descriptive title for the shortened link"
+                placeholder="(Optional)"
+                {...itemProps.title}
+            />
+            {useCustomTags ? (
+                <>
+                    <Form.TextField
+                        title="Custom tags"
+                        info="Comma separated list of custom tags"
+                        placeholder="(Optional)"
+                        storeValue={false}
+                        id={itemProps.customTags.id}
+                        value={itemProps.customTags.value}
+                        onChange={(value) => setValue("customTags", value.replace(" ", "-"))}
+                    />
+                </>
+            ) : (
+                <Form.TagPicker
+                    title="Tags"
+                    info="List of pre-existing tags"
+                    placeholder="(Optional)"
+                    storeValue={false}
+                    {...itemProps.tags}
+                >
+                    {tagsData?.map((tag) => <Form.TagPicker.Item title={tag} key={tag} value={tag} />)}
+                </Form.TagPicker>
+            )}
+            <Form.Checkbox
+                title="Tags input mode"
+                label="Set custom tags instead of selecting from the list"
+                storeValue={false}
+                id="useCustomTags"
+                value={useCustomTags}
+                onChange={setUseCustomTags}
+            />
+            <Form.Separator />
+            <Form.Dropdown title="Domain" info="The domain for the shortened url" storeValue {...itemProps.domain}>
+                {domainsData?.map((domain) => (
+                    <Form.Dropdown.Item title={domain.domain} key={domain.domain} value={domain.domain} />
+                ))}
+            </Form.Dropdown>
+            <Form.TextField
+                title="Custom slug"
+                placeholder="(Optional)"
+                info="The slug to be used instead of the short code"
+                {...itemProps.customSlug}
+            />
+            <Form.TextField
+                title="Short code length"
```

Comment:
```suggestion
                title="Short Code Length"
```
Can you do this with all of the form elements?